src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Terminal42\AutoRegistrationBundle\ContaoManager;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\ManagerPlugin\Bundle\Config\ConfigInterface;
+use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use Terminal42\AutoRegistrationBundle\Terminal42AutoRegistrationBundle;
+
+
+class Plugin implements BundlePluginInterface
+{
+
+    /**
+     * Gets a list of autoload configurations for this bundle.
```

Comment:
Use `{@inheritdoc}` so we don't need to duplicate the description from interface.